### ISSUE TITLE
Add settings to disable comment, mention, and private message notifications

### ISSUE THREAD

**Issue by dessalines** [LemmyNet/lemmy#1883]
We already have one for post notifs, so adding the others should be easy enough. 

Take a vote on what the default should be.

**Comment by aureumleonis** [comment]
I'm taking a look at this as a first issue. Seems like a good spot to get to know the codebase.

**Comment by Shou** [comment]
Yes please. This is one of the super annoying things with Reddit - it shoves reply notifications in your face even if you've disabled all available notification settings. It's a (potentially) hostile feature that can feed social media addictions (compulsions) instead of letting users have control over what the website UI design incentivises you, as a user, to do. You see that little numeric counter and you've been trained for years by all the other software using similar UI elements, to get rid of it, and bring it back to zero - it's not an easy habit to break.

**Comment by wiki-me** [comment]
I vote for enabled by default, makes that feature more easily discoverable.

**Comment by me-cfs** [comment]
An issue that had the idea of letting people mute replies to specific posts or comments was closed in favour of this one. So I just want to say I +1 this feature request.

**Comment by Nutomic** [comment]
https://github.com/LemmyNet/lemmy/pull/5604
